adaptive_components/lib/src/adaptive_column.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+import 'package:flutter/material.dart';
+import 'adaptive_container.dart';
+
+/// The [AdaptiveColumn] follows the Material guideline for responsive grids.
+///
+/// https://material.io/design/layout/responsive-layout-grid.html#columns-gutters-and-margins
+///
+/// This widget intigrates with the [AdaptiveContainer] widget.
+/// The [AdaptiveContainer] widget has a parameter called columns which represents
+/// the amount of columns it should take according to the breakpoints package.
+///
+/// So if the user has 6 adaptive container and each container represents two columns
+/// then the 6 adaptive container would all fit within one Row on a extra large screen
+/// because extra large screens have 12 columns per row.
+///
+/// Learn more about the breakpoint system:
+///
+/// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
+class AdaptiveColumn extends StatelessWidget {
+  /// Creates a vertical array of children. Going from left to right and then
```

Comment:
I think this description needs more. Maybe describing how its different from a regular `Column` in Flutter ?

Reply:
Done